src/test/java/edu/ucsb/cs156/organic/testconfig/MockCurrentUserServiceImpl.java

```diff
@@ -48,8 +48,8 @@ public User getMockUser(SecurityContext securityContext, Authentication authenti
       pictureUrl = "https://example.org/" +  user.getUsername() + ".jpg";
       fullName = "Fake " + user.getUsername();
       emailVerified = true;
-      admin= (user.getUsername().equals("admin"));
-    }
+      admin = false;
```

Comment:
I dont really understand this change

Reply:
The old code was left over from a convention that no longer applies where the admin or not was inferred from the name of the user.  That's no longer the case; there's other code that sets the admin status.